src/uct/cuda/cuda_ipc/cuda_ipc_iface.c

```diff
@@ -318,6 +432,9 @@ static UCS_CLASS_CLEANUP_FUNC(uct_cuda_ipc_iface_t)
     uct_base_iface_progress_disable(&self->super.super,
                                     UCT_PROGRESS_SEND | UCT_PROGRESS_RECV);
     ucs_mpool_cleanup(&self->event_desc, 1);
+    if (self->eventfd != -1) {
```

Comment:
why would eventfd be -1? (unless we create the eventfd on demand)

Reply:
added on demand eventfd creation now